test/controllers/api/v2/content_view_versions_controller_test.rb

```diff
@@ -136,6 +150,15 @@ def test_export_histories_protected
       end
     end
 
+    def test_export_api_status_protected
+      allowed_perms = [@view_permission]
+      denied_perms = [@create_permission, @update_permission, @destroy_permission]
```

Comment:
Should `create` be in allowed perms?

Reply:
You bring a good point. May be it should be `export_content_views` not read

Reply:
Settled for read + export

Reply:
updated